src/Microsoft.AspNet.Cors.Core/EnableCorsAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Cors.Core
+{
+    /// <inheritdoc />
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = false)]
```

Comment:
you mean like the call to GetCustomAttributes(inherit: true) will get it?...i don not think so....probably a functional test in MVC could be useful here... 

I get confused with inherited, but looks like i am not the only one...take a look at the matrix :smile: 
http://blogs.msdn.com/b/jmstall/archive/2013/07/17/attributeusage-inherited-flag.aspx